tracer-core/src/test/java/org/zalando/tracer/EnforceCoverageTest.java

```diff
@@ -1,36 +0,0 @@
-package org.zalando.tracer;
-
-import com.google.gag.annotation.remark.Hack;
-import com.google.gag.annotation.remark.OhNoYouDidnt;
-import org.junit.jupiter.api.Test;
-
-import static org.hamcrest.Matchers.hasToString;
-import static org.hamcrest.Matchers.notNullValue;
-import static org.hamcrest.MatcherAssert.assertThat;
-
-@Hack
-@OhNoYouDidnt
-final class EnforceCoverageTest {
-
-    @Test
-    void shouldUsePhraseGeneratorHolderConstructor() {
-        new PhraseGenerator.Holder();
-    }
-
-    @Test
-    void shouldUseTracerFactoryConstructor() {
-        new TracerFactory();
-    }
-
-    @Test
-    void shouldUseTraceListenersConstructor() {
-        new TraceListeners();
-    }
-
-
-    @Test
-    void shouldCoverTracerBuilderToString() {
-        assertThat(Tracer.builder(), hasToString(notNullValue()));
-    }
```

Comment:
Not a constructor test

Reply:
No it is not, and it also looks useless

Reply:
I.e. removal look like an improvement :)